bigframes/core/compile/scalar_op_compiler.py

```diff
@@ -658,10 +658,6 @@ def isin_op_impl(x: ibis_types.Value, op: ops.IsInOp):
 
 @scalar_op_compiler.register_unary_op(ops.RemoteFunctionOp, pass_op=True)
 def remote_function_op_impl(x: ibis_types.Value, op: ops.RemoteFunctionOp):
-    if not hasattr(op.func, "bigframes_remote_function"):
```

Comment:
This is the wrong layer at which to do this. Please instead call this on the actual Series object.

That way numpy ufuncs will also work as expected.

Reply:
Also, please add tests for passing in numpy ufuncs directly in addition to the existing lambda tests.

Reply:
Added tests with select numpy ufuncs supported by bigframes